cassandra40/src/main/java/org/apache/cassandra/sidecar/cassandra40/StorageJmxOperations.java

```diff
@@ -0,0 +1,18 @@
+package org.apache.cassandra.sidecar.cassandra40;
+
+import java.util.Map;
+
+/**
+ * An interface that pulls a method from the Cassandra Storage Service Proxy
+ */
+interface StorageJmxOperations
```

Comment:
Why it is in `cassandra40`? Do you expect it to be different in a different cassandra release?

Reply:
yeah, the interface evolves with versions, so I expect this to change in 41 and most likely the interface evolved and it looks different in 30, 31 and so on